src/components/DashboardAppBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const DashboardAppBar = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const logout = () => {
+    firebase.logout().then(() => {
+      navigate('/')
+    })
+  }
+  return (
+    <div>
+      <AppBar position="static">
```

Comment:
Do you need that `position="static"`? `static` is the default value for that [CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/position#values), so it's not changing anything, I believe.

Reply:
Without it other content starts from the top of the page, but it should start from the end of header.

Reply:
Interesting, but makes sense.